### DISCUSSION TITLE
pip install <omz_dir>/demos/common/python says "The '<' operator is reserved for future use."

### DISCUSSION THREAD

**Discussion by AmrInCube** [community/community#67510] [Programming Help] [answered]
Hi, I have been trying to teach myself about OpenVINO and OMZ (Open Model Zoo) and I have run into an issue with this line in my command prompt. I copied this command off of the readme.md file from https://github.com/openvinotoolkit/open_model_zoo/tree/master/demos/common/python/openvino/model_zoo/model_api 

I was trying this in a powershell terminal and from googling it says that i shouldnt do this. It says that i should try it in my cmd terminal and itll be fine but when i do that it says "The system cannot find the file specified". Obviously im probably doing something quite nooby, im not a very experienced coder. Any help in how to get this installation would be great. I believe I am getting import issues due to not having this installed. I am currently on the 'Live Human Pose Estimation with OpenVINO™' online tutorial using Jupyters Notebook. The link to the tutorial can be found here: https://docs.openvino.ai/2023.0/notebooks/402-pose-estimation-with-output.html

I am looking into this because my first block of text from the tutorial is giving me an error, initially it was about not having 'decoder' installed so i installed it but now it says "Import Error: 'Cannot import name 'OpenPoseDecoder' from 'decoder'" followed by file path to ...\Python311\site-packages\decoder\___init__.py

I have attached my code but if im not mistaken it is the same code that is in the tutorial up to the "run_pose_estimation(source=0, flip=True, use_popup=False)" line.

If anyone knows any answer to this issue help would be greatly appreciated. 
[LiveHumanPoseEstimation.zip](https://github.com/community/community/files/12651842/LiveHumanPoseEstimation.zip)

**Comment by seyLu** [comment, marked as answer]
#### Hi @AmrInCube 

Assuming you've already installed `openvino` via pip:
```bash
pip install openvino
```

The next step:
> Installing Python* Model API package
Use the following command to install Model API from source:
`pip install <omz_dir>/demos/common/python`

tells you to build from source since the lib is not published on Pypi. 

You can do this by cloning the repo and then substitute `<omz_dir>` with the file path of the cloned repo.

```bash
git clone https://github.com/openvinotoolkit/open_model_zoo.git

# substitute <omz_dir> with path to omz_dir
pip install <omz_dir>/demos/common/python
``` 

Pip will look for `setup.py` in `<omz_dir>/demos/common/python` and install the package:
![image](https://github.com/community/community/assets/98249191/7a2641ef-45db-415e-a605-056ed29fad3e)

**Reply by AmrInCube** [reply]
Thank you so much for this help <3
